jobs/tests/fixtures_schemas.py

```diff
@@ -59,3 +59,23 @@ def mock_AnnotationJobParams2():
         deadline=datetime.datetime.utcnow() + datetime.timedelta(days=1),
     )
     return mockAnnotationJobParams2
+
+
+
+@pytest.fixture
+def mock_Extraction_AnnotationJobParams():
+    mockExtractionAnnotationJobParams = schemas.ExtractionWithAnnotationJobParams(
+        name="MockExtractionWithAnnotationJob",
+        files=[1, 2],
+        datasets=[1, 2],
+        pipeline_name="1",
+        pipeline_engine="Airflow",
+        annotators=["annotator1", "annotator2"],
+        validators=["validator1", "validator2"],
+        owners=["owner1"],
+        categories=["category1", "category2"],
+        is_auto_distribution=False,
+        deadline=datetime.datetime.utcnow() + datetime.timedelta(days=1),
+        validation_type="cross",
+    )
+    return mockExtractionAnnotationJobParams
```

Comment:
1. no need for extra var, just return that
2. pytest recommends yielding from fixtures (https://docs.pytest.org/en/stable/how-to/fixtures.html#yield-fixtures-recommended)
3. add a new line at the end of the file